app/src/app/api/bridging-fee/route.ts

```diff
@@ -0,0 +1,34 @@
+import { CACHE_REVALIDATE_IN_SECONDS } from '@/services/balance'
+import { getErrorMessage } from '@/utils/transferTracking'
+import { getParaEthTransferFees } from '@paraspell/sdk'
+import { unstable_cache } from 'next/cache'
+import { NextResponse } from 'next/server'
+
+/**
+ * Fetch the AH -> Ethereum bridging fee
```

Comment:
8:18  Warning: tsdoc-escape-greater-than: The ">" character should be escaped using a backslash to avoid confusion with an HTML tag  tsdoc/syntax
```suggestion
 * Fetch the AH -\> Ethereum bridging fee
```